src/selectors/__tests__/getPrevRank.ts

```diff
@@ -1,20 +1,70 @@
-import { initialState, reducerFlow } from '../../util'
-import { getAllChildren } from '../../selectors'
-import { newSubthought, newThought, cursorBack } from '../../reducers'
+import { initialState, isFunction, reducerFlow } from '../../util'
+import { RANKED_ROOT } from '../../constants'
+import { getAllChildrenSorted } from '../../selectors'
+import { importText, newSubthought, newThought } from '../../reducers'
 import getPrevRank from '../getPrevRank'
 
-it('getPrevRank when all the thoughts in the context are hidden', () => {
+it('get rank above all children', () => {
```

Comment:
@shresthabijay FYI You can exclude the module name from the individual tests since it is automatically included in the Jest output.